test/browser_tests/page_objects/page_blog.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var _mixin = require( '../utils/mixin.js' ).mixin;
+var filter = require( '../shared_objects/filter.js' );
+
+function Blog() {
+
+  _mixin( this, filter );
+
+  this.get = function() {
+    browser.get( '/blog/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.contentSidebar = element( by.css( '.content_sidebar' ) );
+
+  this.popularStories =
+  this.contentSidebar.element( by.css( '.popular-stories' ) );
+
+  this.popularStoriesTitle =
+  this.popularStories.element( by.css( '.header-slug_inner' ) );
+
+  this.stayInformedSection = this.contentSidebar
+  .element( by.css( '[data-qa-hook="stay-informed-section"]' ) );
+
+  this.stayInformedSectionTitle =
+  this.stayInformedSection.element( by.css( '.header-slug_inner' ) );
+
+  this.emailSubscribeForm =
+  this.stayInformedSection.element( by.css( '#email-subscribe-form' ) );
+
+  this.emailSubscribeForm =
+  this.stayInformedSection.element( by.css( '#email-subscribe-form' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/f18474a240fd0f56bdd225730736293f2f28c11c